src/utils/decorators/input-validation/__spec__.js

```diff
@@ -1213,11 +1213,13 @@ describe('InputValidation', () => {
 
       it('returns an info icon', () => {
         expect(instance.validationHTML[0].props.type).toEqual('info');
+        // TODO - 1007 - find not returning element though it is printed in the html()
         expect(wrapper.find('.common-input__icon.common-input__icon--info').exists()).toBeTruthy();
       });
 
       it('returns a div for the info message', () => {
-        expect(wrapper.find('.common-input__message-wrapper').exists()).toBeTruthy();
+        // TODO - 1007 - find not returning element though it is printed in the html()
+        expect(wrapper.find('.common-input__message-wrapper').exists()).toEqual(true);
```

Comment:
again, there is another method used in this spec to check the class name (checking the prop rather than using enzyme).